spec/validate.py

```diff
@@ -232,6 +244,19 @@ def validate_view(path):
     return data
 
 
+def validate_analyzer(path):
+    """Validate ArangoDB analyzer config"""
+    print(f"  validating {path}..")
```

Comment:
this is completely pedantic, but can you change this to be `{path}...` (i.e. three dots, an ellipsis)?